DatadogLogs/Sources/Logs.swift

```diff
@@ -0,0 +1,113 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import DatadogInternal
+
+/// iOS Log Collection
+///
+/// Send logs to Datadog from your iOS applications with Datadog’s dd-sdk-ios client-side logging library and leverage the following features:
+/// - Log to Datadog in JSON format natively.
+/// - Use default and add custom attributes to each log sent.
+/// - Record real client IP addresses and User-Agents.
+/// - Leverage optimized network usage with automatic bulk posts.
+public struct Logs {
+    /// The Logs general configuration.
+    ///
+    /// This configuration will be applied to all Logger instances.
+    public struct Configuration {
+        public typealias EventMapperClosure = (LogEvent) -> LogEvent?
```

Comment:
For consistency with `RUM*EventMapper`:
```suggestion
        public typealias LogEventMapper = (LogEvent) -> LogEvent?
```

Reply:
I think it conflicts with a public protocol available for cross platform, let me double check.

Reply:
Resolved by your suggestion above 👍